airbyte-integrations/connectors/source-amplitude/acceptance-test-config.yml

```diff
@@ -1,24 +1,35 @@
 connector_image: airbyte/source-amplitude:dev
-tests:
+test_strictness_level: high
+acceptance_tests:
   spec:
-    - spec_path: "source_amplitude/spec.json"
-      backward_compatibility_tests_config:
-        disable_for_version: "0.1.17"
+    tests:
+      - spec_path: "source_amplitude/spec.json"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
+    tests:
+      - config_path: "secrets/config.json"
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/streams_with_output_records_catalog.json"
+    tests:
+      - config_path: "secrets/config.json"
+        expect_records:
+          path: "integration_tests/expected_records.txt"
+        empty_streams:
+          - name: "events"
+            bypass_reason: "The stream is too large, skipped."
+          - name: "annotations"
+            bypass_reason: "Stream has no records"
```

Comment:
What'd be the effort to seed this stream with data?

Reply:
Done.